src/App.css

```diff
@@ -0,0 +1,547 @@
+@import "https://fluidcssv2.netlify.app/styles.css";
```

Comment:
Instead of keeping all your CSS in one place maybe you can split the code into modules. just a suggestion!